03LessonThree/in-class-project/daily-planet.html

```diff
@@ -5,6 +5,28 @@
         <link rel="stylesheet" href="css/daily-planet-style.css"/>
     </head>
     <body>
-        <!-- You code start here -->
+      <header>
+        <h1>Daily Planet</h1>
+          <nav><a href="http://espn.go.com">Sports</a></nav>

```

Comment:
When we start using Bootstrap, this will actually make THREE nav bars! Instead, have a nav tag, and nest an unordered list inside.